providers/amazon/tests/unit/amazon/aws/hooks/test_sns.py

```diff
@@ -90,3 +90,28 @@ def test_publish_to_target_error(self):
                     "test-non-iterable": object(),
                 },
             )
+
+    def test_publish_to_target_with_deduplication(self):
+        hook = SnsHook(aws_conn_id="aws_default")
+
+        message = "Hello world"
```

Comment:
You could consider moving variables that are used repeatedly (like this one) to constants at the top of the file instead but other than that, looks good!